src/main/java/org/littil/api/contact/repository/ContactEntity.java

```diff
@@ -0,0 +1,26 @@
+package org.littil.api.contact.repository;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.littil.api.auditing.repository.AbstractAuditableEntity;
+import org.littil.api.user.repository.UserEntity;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity(name = "Contact")
+@Table(name = "contact")
+public class ContactEntity extends AbstractAuditableEntity {
```

Comment:
De userEntity van de sender ontbreekt hier.

Reply:
de sender == createdBy; de aanmaker is altijd de sender.